FROST-Server.Core.Model/src/main/java/de/fraunhofer/iosb/ilt/frostserver/path/UrlHelper.java

```diff
@@ -45,7 +49,11 @@ public static String generateNextLink(ResourcePath path, Query query) {
         return nextLink;
     }
 
-    public static String generateSelfLink(String serviceRootUrl, Version version, EntityType entityType, Object id) {
+    public static String generateSelfLink(Query query, String serviceRootUrl, Version version,
+            EntityType entityType, Object id) {
```

Comment:
I am trying to match the existing style but FROST-Server is not really friendly with contributors with its custom java style that seems to be defined in https://github.com/FraunhoferIOSB/FROST-Server/blob/master/nb-configuration.xml This NetBeans configuration is hard to use in other editors. I am using Eclipse and I could not find how to use this NetBeans  config.
It would be easier to use a more common style such as https://google.github.io/styleguide/javaguide.html

Reply:
I fixed it.

Reply:
It's using the standard java formatting rules Sun published back in 1997.
For some reason Eclipse manages to have no setting to exactly follow those, even though Eclipse has a million configuration settings for formatting...